sway/tree/container.c

```diff
@@ -418,6 +418,32 @@ void container_for_each_child(struct sway_container *container,
 	}
 }
 
+struct sway_container *container_obstructing_fullscreen_container(struct sway_container *container)
+{
+	if (!container) {
```

Comment:
Most of the functions here don't do nullchecks internally, let's just move this check to the caller for consistency.

Reply:
Updated